contracts/staking/ClusterRewards.sol

```diff
@@ -220,32 +215,6 @@ contract ClusterRewards is
 
     //-------------------------------- Admin functions end --------------------------------//
 
-    //-------------------------------- User functions start --------------------------------//
-
-    function feed(bytes32 _networkId, address[] calldata _clusters, uint256[] calldata _payouts, uint256 _epoch) external onlyFeeder {
-        require(receiverStaking.START_TIME() + SWITCHING_PERIOD + 1 days > block.timestamp, "CRW:F-Invalid method");
-        uint256 rewardDistributed = rewardDistributedPerEpoch[_epoch];
```

Comment:
Rename all variables which aren't used anymore with `unused` like  we do  in other contracts.

Reply:
removed